app/editor/src/features/reports/av-overview/OverviewGrid.tsx

```diff
@@ -85,7 +85,11 @@ export const OverviewGrid: React.FC<IOverviewGridProps> = ({ editable = true, in
         sort: ['publishedOn asc'],
       }).then((data) => {
         setContentItems(data.items);
-        const newClips = data.items.map((c) => new OptionItem(c.headline, c.id)) as IOptionItem[];
+        const newClips = data.items.map((c) => {
```

Comment:
Does this fix also need to be applied to the Subscriber site?

Reply:
I will check. thanks!

Reply:
Verified the report looks good in Subscriber.
Thanks!